isotovideo

```diff
@@ -172,12 +172,24 @@ sub signalhandler_chld {
 sub init_backend {
     my ($name) = @_;
     $bmwqemu::vars{BACKEND} ||= "qemu";
-    my $dir = getcwd;
-    # make sure the needles are initialized before the backend process is started
     no autodie qw(system);
-    my $res = system("rsync -avP --delete --exclude=.git $bmwqemu::vars{PRODUCTDIR}/needles/ $dir/needles/");
-    die "Failed to rsync needles: $@" if $res;
-    needle::init("$dir/needles");
+
+    #We want to use the pool directory instead of the
+    # cwd (which would be the worker's directory)
+    # to be able to share the needles between workers
+    my $shared_needle_dir = getcwd() . "/..";
```

Comment:
can we then please also make the whole syncing behaviour optional configured over the workers ini? At least we could make the old behaviour optional. Maybe even make the sync call itself configurable.